cmd/generate.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright (C) 2021-2025, Kubefirst
```

Comment:
Used Kubefirst for consistency with rest of the project - there's a task to update everything to `Konstruct Inc`, but outside of this work.